mla/test_statistics.py

```diff
@@ -130,9 +130,6 @@ def _i3_ts(sob: np.ndarray, prepro: Preprocessing,
     if prepro.n_events == 0:
         return 0
 
-    else:
-        ns_ratio = _calculate_ns_ratio(sob)
```

Comment:
The ns_ratio line still needs to be there, just not in an else

Reply:
My bad,just removed by mistake